package.json

```diff
@@ -7,6 +7,7 @@
     "description": "specifies dependencies needed for developing taco",
     "version": "0.0.1",
     "devDependencies": {
+        "taco-simulate": "^0.1.1",
```

Comment:
Hmmm... That was something I added right at the very beginning and for the life of me I can't recall why. I'll remove.